README.md

```diff
@@ -114,6 +112,15 @@ $ bundle exec pry
     kind="compute#instance",
 ```
 
+## Supported Ruby Versions
```

Comment:
This makes sense, and reads well, but I'd put it as an h3 under Important Notices at the top, as this is pretty far down the page.

Reply:
Ack, added a notice. Don't want to put the whole paragraph up there though, so created a link.